CHANGES.md

```diff
@@ -29,6 +29,8 @@ Format of the entries must be.
 
 * DC/OS UI X-Frame-Options value can be configured (DCOS-49594)
 
+* Telegraf's Mesos input plugin now uses the configured timeout value for http client timeouts instead of sending a 'timeout' query param to the Mesos metrics endpoint (DCOS-50672)
```

Comment:
This has a lot of internal detail that wouldn't be meaningful to an end-user. How about: "Telegraf cancels timed out requests for internal Mesos metrics, instead of collecting partial metric data."